packages/utilities/src/dom/getDocument.ts

```diff
@@ -0,0 +1,13 @@
+import { _isSSR } from './setSSR';
+/**
+ * Helper to get the document object.
```

Comment:
While you're here, can you add a note in the comment that this helper exists to help support server-side rendering (where `document` is undefined)? Plus a bit about whatever the reason is for the `ownerDocument` check.